server/public/model/report.go

```diff
@@ -75,6 +82,43 @@ type UserReport struct {
 	UserPostStats
 }
 
+func (u *UserReport) ToReport() []string {
+	lastStatusAt := ""
+	if u.LastStatusAt != nil {
+		lastStatusAt = time.UnixMilli(*u.LastStatusAt).String()
+	}
+	lastPostDate := ""
+	if u.LastPostDate != nil {
+		lastStatusAt = time.UnixMilli(*u.LastPostDate).String()
```

Comment:
Bug: should be assigning to `lastPostDate`, not `lastStatusAt`.

Reply:
Whoops that'll do it lol